lizrd/scripts/grid.py

```diff
@@ -113,6 +115,11 @@
         job_name = training_args["name"]
         training_args["n_gpus"] = setup_args["n_gpus"]
 
+        singularity_env_arguments = make_singularity_env_arguments(
+            hf_datasets_cache_path=setup_args["hf_datasets_cache"],
+            neptune_key=args.neptune_key,
+        )
+
```

Comment:
one either dies a "i write long, transprarent files" hero
or lives long enough to become a JS-like villian